extensions/webview/libwebview_toolbar.m

```diff
@@ -1316,6 +1331,87 @@ static int displayMode(lua_State *L) {
     return 1 ;
 }
 
+/// hs.webview.toolbar:toolbarStyle([style]) -> toolbarObject
+/// Method
+/// Get or set the toolbar's style.
+///
+/// Parameters:
+///  * style - an optional string to set the style of the toolbar to "automatic", "expanded", "preference", "unified", or "unifiedCompact".
+///
+/// Returns:
+///  * if an argument is provided, returns the toolbar object; otherwise returns the current value
+///
+///  Notes:
+///   * This is only available for macOS 11.0+. Will return `nil` if getting on an earlier version of macOS.
+///   * `automatic` - A style indicating that the system determines the toolbar’s appearance and location.
+///   * `expanded` - A style indicating that the toolbar appears below the window title.
+///   * `preference` - A style indicating that the toolbar appears below the window title with toolbar items centered in the toolbar.
+///   * `unified` - A style indicating that the toolbar appears next to the window title.
+///   * `unifiedCompact` - A style indicating that the toolbar appears next to the window title and with reduced margins to allow more focus on the window’s contents.
+static int toolbarStyle(lua_State *L) {
+    LuaSkin *skin = [LuaSkin sharedWithState:L] ;
+    [skin checkArgs:LS_TUSERDATA, USERDATA_TB_TAG, LS_TSTRING | LS_TOPTIONAL, LS_TBREAK] ;
+    HSToolbar *toolbar = [skin toNSObjectAtIndex:1] ;
+        
+    if (lua_gettop(L) == 2) {
+        if (@available(macOS 11.0, *))
+        {
+            NSString *type = [skin toNSObjectAtIndex:2] ;
+            BOOL valid = NO;
+            if ([type isEqualToString:@"automatic"]) {
+                valid = YES;
+                toolbar.toolbarStyle = NSWindowToolbarStyleAutomatic ;
+            } else if ([type isEqualToString:@"expanded"]) {
+                valid = YES;
+                toolbar.toolbarStyle = NSWindowToolbarStyleExpanded ;
+            } else if ([type isEqualToString:@"preference"]) {
+                valid = YES;
+                toolbar.toolbarStyle = NSWindowToolbarStylePreference ;
+            } else if ([type isEqualToString:@"unified"]) {
+                valid = YES;
+                toolbar.toolbarStyle = NSWindowToolbarStyleUnified ;
+            } else if ([type isEqualToString:@"unifiedCompact"]) {
+                valid = YES;
+                toolbar.toolbarStyle = NSWindowToolbarStyleUnifiedCompact ;
+            } else {
+                return luaL_error(L, "invalid toolbarStyle: '%s'", [type UTF8String]) ;
+            }
+            // Update the toolbar if it's already visible:
+            if (valid && toolbar.windowUsingToolbar) {
+                toolbar.windowUsingToolbar.toolbarStyle = toolbar.toolbarStyle;
+            }
+            lua_pushvalue(L, 1) ;
```

Comment:
if the method isn't going to throw an error when used pre macOS 11.0 (and I don't think that it should), then the pushvalue should be moved out of the guarded block.